content/release/ReleaseNotes365.md

```diff
@@ -0,0 +1,52 @@
+* Attempting to approve or deny pending vulnerabilities caused an error.
+* Users with an Edit role couldn't export library data from the library's **Overview** page.
+* Sending a vulnerability to Jira with an attachment caused an error in some cases.
+* Managing user access groups caused internal server errors for some SaaS users.
+* Cache issues prevented users from seeing vulnerability details after applying a license to an application.
+* Slack integration notifications ignored New Asset selections made in **Notification Settings**.
+* The **Libraries** grid timed out and returned errors in some users' SaaS environments.
+
+## Improvements
+
+* Set up [session metadata](user-vulnerableapps.html#session) to pinpoint the source of vulnerabilities for each of your applications. To get started, configure your agent to report one of the available metadata types, including build numbers, branch names, repositories, and committers. Go to your application’s **Vulnerabilities** tab to see the data in the new vulnerabilities timeline, and use the **Seen By** column in the grid to filter vulnerabilities by specific values.
+
+* Use [source names](admin-ipmgmt.html#source-name) to label attack events by expected sources so you can promptly choose which attack events to investigate. All you need is the IP information for a known source (like a pen tester) to get started. When you view attacks in the **Attacks > Monitor** page and **Attacks Details** pages, Contrast will display the source name instead of the attacker’s IP information. (We’ll be adding source names to the Attack Events grid next!)
+
+* Contrast offers two new Protect rules against unsafe files: The Unsafe File Uploads rule blocks malicious files being uploaded to web applications, and the Zip File Overwrite rule protects against malicious files and directory structures within zip files uploaded to web applications. These rules available for all languages.
+
+
+## Agent Updates
+
+### Java summary
+
+Check back next release for updates!
+
+### .NET summary 
+
+The .NET team added support for agent use within [Docker containers](installation-netinstall.html#net-docker) for .NET 4.5.2+ applications. We improved accuracy of an insecure authentication protocol Assess rule, and added a new Assess rule to detect the “X-Powered-By” header. We fixed a bug that caused a null reference error in processing exclusions. We also added support for setting `application.code`.
+
+### .NET Core summary
+
+The [.NET Core agent](installation-netcore.html#netcore-overview) for Windows is now available! The agent supports the same expansive Assess and Protect security policy as the .NET Framework agent, just for your .NET Core applications. To start using the agent, check out the [system requirements](installation-netcore.html#netcore-system), and then [download and install](installation-netcoreinstall.html) the agent from the Contrast UI. 
```

Comment:
The .NET Core agent currently has a slightly smaller policy / set of supported rules than the full .NET Framework agent. (We're working to fill in the gaps...) I'm not sure the best way to word this but I'd be comfortable saying that the .NET Core agent supports most of the same policy as the .NET framework agent, including detection of all of the most important vulnerabilities and attacks.